src/lib.rs

```diff
@@ -432,16 +479,24 @@ pub struct Thread<BlockNumber, Moment, AccountId> {
     /// to a category when it was added.
     /// Is needed to give light clients assurance about getting all threads in a given range,
     /// `created_at` is not sufficient.
-    thread_nr: u32,
+    /// Starts at 1 for first thread in category.
+    thread_nr_in_category: u32,
```

Comment:
Regarding `_nr_`
I have checked the Internet for "what is a shortcut for a number", and the answer is `no`.
https://forum.wordreference.com/threads/abbreviation-of-number-n-n%C2%B0-nr-nbr-no.264328/
https://english.stackexchange.com/questions/43095/what-is-the-correct-abbreviation-for-the-word-numbers

but I guess that `no` could be considered as "No" word (negative).

Reply:
What about `sn_in_category` or something like this. Which means `a serial number of the thread in its category.`

Reply:
At least I believe that `thread_` prefix could be removed from `thread_nr_in_category` because this field is already in a struct called `Thread`.

Reply:
I removed `thread_` prefix and also `post_nr`.